docs/development_suite/api-console/api-design/mongo_views.md

```diff
@@ -11,11 +11,6 @@ sidebar_label: Create Views
 
 ## Create a new View
 
-:::danger
```

Comment:
If I use a v7 version prior the fix I still can't use the feature, wouldn't it be better to put this as a lower level warning stating the actual version that do not support views over encrypted fields?

Reply:
Is `info` level sufficient?

Reply:
In the suggestion I converted it in an info banner by specifying the range of versions the does not support the feature. Do you think that would be okay?